README.md

```diff
@@ -204,7 +163,56 @@ Matrix elements are delimited by colons (`:`)
 | --------------------- | ------------------------------- |
 | `VAGRANT_BOX_DIR`     | Path to top of Vagrant box tree |
 | `SIMP_ISO_JSON_FILES` | List of absolute paths/globs to SIMP ISO `.json` files to consider (delimiters: `:`, `,`).  This variable can be used as an alternative to the matrix entry `json=`.  Non-existent paths will be discarded with a warning message |
-| `MATRIX_LABEL`        | Label for this matrix run that will prefix each iteration's directory name (default: `build`) |
+| `MATRIX_LABEL`        | Label for this matrix run that will prefix each iteration's directory name (default: `build_<YYYYmmdd>_<HHMMSS>`) |
+
+
+### Using `rake simp:packer:build`
+
+```sh
+TMPDIR=/some/tmp/dir \
+  [ENV_VAR=value ...] \
+  bundle exec rake simp:packer:build[vars_json,packer_yaml,simp_conf_yaml,test_dir]
```

Comment:
What are the arguments to this build task?  Do all need to be specified?  Previously only the directory needed to be specified.

Reply:
I fixed the README